packages/run-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -292,15 +293,16 @@ const machineBehavior = {
         debtMint.burnLosses(harden({ RUN: toMint }), mintSeat);
         throw e;
       } finally {
+        // Not that if this assertion may fail because of an error in the
```

Comment:
If this correction isn't right, please reword.
```suggestion
        // Note that if this assertion may fail because of an error in the
```